libs/filamat/src/GLSLPostProcessor.cpp

```diff
@@ -169,6 +169,7 @@ void GLSLPostProcessor::spirvToMsl(const SpirvBlob *spirv, std::string *outMsl,
     }
 
     mslOptions.argument_buffers = true;
+    mslOptions.ios_support_base_vertex_instance = true;
```

Comment:
Just curious, why is this required?

Reply:
Without this, the multiview shader doesn't compile for iOS, complaining `BaseInstance requires Metal 1.1 and Mac or Apple A9+ hardware` from spirv-cross.

https://github.com/KhronosGroup/SPIRV-Cross/issues/1612